LCM/scripts/python3/PerformInventory.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python3
+from fcntl                import flock, LOCK_EX, LOCK_UN, LOCK_NB
+from imp                  import load_source
+from os                   import listdir, system
+from os.path              import dirname, isfile, join, realpath
+from shutil               import move
+from subprocess           import Popen, PIPE
+from sys                  import argv, exc_info, exit, stdout, version_info
+from traceback            import format_exc
+from xml.dom.minidom      import parse
+from OmsConfigHostHelpers import write_omsconfig_host_telemetry, write_omsconfig_host_switch_event, write_omsconfig_host_log, stop_old_host_instances
```

Comment:
get someone from CxP to test

Reply:
Will be done as a part of their signoff.